### ISSUE TITLE
Bug when using SQL connection

### ISSUE THREAD

**Issue by jmoneste** [mRemoteNG/mRemoteNG#502]
|||
|--:|---|
|Operating system | Windows 2008 R2 |
|mRemoteNG version| 1.74.6023.15437 |
|SQL version| MS SQL 2008R2 |

When connected to SQL and duplicating a Container (CTRL +D) to move it to another container the ParentID is also duplicated which causes that on reload all the containers are mixed into the original container.
Example:

Parent 0
  - Container 1 (on Parent 0)
  - - - Container 2
  - - - - - Container 3  (Duplicate this one to create Container 6 and move to Container 5)
  - Container 4 (on Parent 0)
   - - - Container 5

Expected result:

Parent 0
  - Container 1 (on Parent 0)
  - - - Container 2
  - - - - - Container 3 
  - Container 4 (on Parent 0)
  - - - Container 5
  - - - - - Container 6

After hitting load (or closing and reopen) what I get is this:
As many as you duplicate, they will always be listed under 2.

Parent 0
  - Container 1 (on Parent 0)
  - - - Container 2
  - - - - - Container 3 
  - - - - - Container 6
  - Container 4 (on Parent 0)
  - - - Container 5

Since I have a lot of duplicated entries it becomes a nightmare, I had to manually update the ParentID using update statements.

I have attached an screenshot of how it looks like.

![issue](https://cloud.githubusercontent.com/assets/26974441/24914470/89b03a02-1e91-11e7-91eb-77294862e061.png)

**Comment by kmscode** [comment]
Looks like it's related to some of the SQL type problems reported in #436 

Potential fixes being worked on in https://github.com/mRemoteNG/mRemoteNG/tree/436_load_coninfo_from_sql

**Comment by jmoneste** [comment]
Thank you so much. I will give it a try.

**Comment by kmscode** [comment]
@jmoneste Nothing to try yet... Some code was checked in, to that branch which will potentially fix your issue, but it hasn't been fully tested yet (and might require some other tweaks based on other things that were uncovered recently with the SQL related code).

**Comment by kmscode** [comment]
@jmoneste Looks like this bug may have been addressed in previous commits. It's possible that you might not encounter this with ["hotfix4"](https://github.com/mRemoteNG/mRemoteNG/releases). However, I have tested this against the unreleased "hotfix5" release that is being put together, and that release should definitely resolve this problem per my testing. Hoping to have that fully released within the next week or so.
